isis/tests/CSMCameraTests.cpp

```diff
@@ -250,6 +251,49 @@ TEST_F(CSMCameraSetFixture, EmissionAngle) {
 }
 
 
+TEST_F(CSMCameraFixture, getParameterIndices) {
+  std::vector<int> paramIndices = {0, 1, 2};
+  EXPECT_CALL(mockModel, getNumParameters())
+      .Times(1)
+      .WillOnce(::testing::Return(3));
+  EXPECT_CALL(mockModel, getParameterType(0))
+      .Times(1)
+      .WillOnce(::testing::Return(csm::param::REAL));
+  EXPECT_CALL(mockModel, getParameterType(1))
+      .Times(1)
+      .WillOnce(::testing::Return(csm::param::REAL));
+  EXPECT_CALL(mockModel, getParameterType(2))
+      .Times(1)
+      .WillOnce(::testing::Return(csm::param::REAL));
```

Comment:
The mock is a little weird here because GeometricModel::getParameterSetIndices is not virtual so we can't mock it out. We have to mock out the functions it calls. See

https://github.com/sminster/csm/blob/master/GeometricModel.cpp#L58